README.md

```diff
@@ -1,70 +1,62 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Math-Magicians
 
-## Available Scripts
+This is a Single Page App (SPA) that allows users to make simple calculations and read a random math-related quotes. The aim of this project is to learn how to use React.
 
-In the project directory, you can run:
+## Live Demo
 
-### `npm start`
+[link to live demo]()
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+> Math-magicians: started of by building the core functionality, which is a calculator that contains a simple UI and is able to perform all math operations.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Built With
 
-### `npm test`
+- Create React App (CRA)
+- Gitflow
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Getting Started
 
-### `npm run build`
+### Clone this repository
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+```bash
+$ git clone git@github.com:Favourezeugwa/math-magicians.git
+$ cd math-magicians
+```
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### Run project
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+```bash
+$ npm start
+```
```

Comment:
- You've added the code to run the project, how would a user install all the packages in your project?
Please, kindly add the `npm install` command also.